mapbox_vector_tile/__init__.py

```diff
@@ -1,25 +1,36 @@
+from typing import Dict, Optional
+
 from mapbox_vector_tile import decoder, encoder
+from mapbox_vector_tile.options import DEFAULT_DECODE_OPTIONS, DEFAULT_ENCODE_OPTIONS
+from mapbox_vector_tile.typing import DecodeOptionsType, EncodeOptionsType
 
 
-def decode(tile, y_coord_down=False):
-    vector_tile = decoder.TileData()
-    message = vector_tile.get_message(tile, y_coord_down)
+def decode(
+    tile,
+    layer_options: Optional[Dict[str, DecodeOptionsType]] = None,
+    default_options: DecodeOptionsType = DEFAULT_DECODE_OPTIONS,
+):
```

Comment:
Now, both decode and encode have the same signature. They take a dictionary which defines layer options and another which defines the default options. The default options are used for layers without entry in the `layer_options` dictionary. The values of `default_options` will also be used to fill the undefined values of any `layer_options` value.